freqtrade/exchange/exchange.py

```diff
@@ -1508,6 +1514,33 @@ def get_historic_trades(self, pair: str,
             self._async_get_trade_history(pair=pair, since=since,
                                           until=until, from_id=from_id))
 
+    def get_funding_fees(self, pair: str, since: Union[datetime, int]) -> float:
+        """
+            Returns the sum of all funding fees that were exchanged for a pair within a timeframe
+            :param pair: (e.g. ADA/USDT)
+            :param since: The earliest time of consideration for calculating funding fees,
+                in unix time or as a datetime
+        """
+
+        self.ccxt_implementation_check("get_funding_fees")
+
+        if type(since) is datetime:
+            since = int(since.strftime('%s'))
+
+        try:
+            funding_history = self._api.fetch_funding_history(
+                pair=pair,
+                since=since
+            )
```

Comment:
Why are you checking for a method called `get_funding_fees` - and then call `fetch_funding_fees` ? 

this makes the whole check kinda pointless ...